code/game/turfs/simulated/floor/plating.dm

```diff
@@ -104,7 +104,7 @@
 		broken = FALSE
 		update_icon()
 
-/turf/simulated/floor/plating/proc/remove_plating(mob/user)
+/turf/simulated/floor/plating/remove_plating(mob/user)
```

Comment:
Why was this not a turf-level proc already?

Reply:
Can ask that for a lot of turf specific procs :P